runtime/dart_isolate.cc

```diff
@@ -858,6 +858,11 @@ void DartIsolate::AddIsolateShutdownCallback(const fml::closure& closure) {
 }
 
 void DartIsolate::OnShutdownCallback() {
+  tonic::DartState* state = tonic::DartState::Current();
+  if (state != nullptr) {
```

Comment:
This seems like maybe it should be an assert not an if?  Is it valid to not have a current dart state but to be getting shutdown callbacks?

Reply:
Done